test_util/test_azure.py

```diff
@@ -167,9 +157,10 @@ def main():
             public_agent_list=dcos_resource_group.get_public_ips(),
             test_cmd=options.test_cmd)
     if result == 0:
-        log.info('Test successsful!')
+        log.info('Test successsful! Deleting Azure resource group')
+        dcos_resource_group.delete()
     else:
-        logging.warning('Test exited with an error')
+        logging.warning('Test exited with an error; resource group will remain for 2 hours')
```

Comment:
s/; resource group will remain for 2 hours/. Resource group preserved for troubleshooting. See https://github.com/mesosphere/cloudcleaner project for cleanup policies/